crates/types/src/net/replicated_loglet.rs

```diff
@@ -168,13 +179,84 @@ impl Appended {
 }
 
 // ** GET_TAIL_INFO
-#[derive(Debug, Clone, Serialize, Deserialize)]
+#[derive(Debug, Clone, Serialize, Deserialize, bilrost::Message)]
 pub struct GetSequencerState {
+    #[bilrost(1)]
     pub header: CommonRequestHeader,
+    #[bilrost(2)]
     pub force_seal_check: bool,
 }
 
-#[derive(Debug, Clone, Serialize, Deserialize)]
+#[derive(Debug, Clone, Serialize, Deserialize, bilrost::Message)]
 pub struct SequencerState {
+    #[bilrost(1)]
     pub header: CommonResponseHeader,
 }
+
+mod dto {
+    use super::*;
+
+    #[derive(Debug, Clone, Serialize, Deserialize, bilrost::Oneof, bilrost::Message)]
+    pub enum SequencerStatus {
+        Unknown,
+        #[bilrost(1)]
+        Ok(()),
+        #[bilrost(2)]
+        Sealed(()),
+        #[bilrost(3)]
+        Gone(()),
+        #[bilrost(4)]
+        LogletIdMismatch(()),
+        #[bilrost(5)]
+        UnknownLogId(()),
+        #[bilrost(6)]
+        UnknownSegmentIndex(()),
+        #[bilrost(7)]
+        NotSequencer(()),
+        #[bilrost(8)]
+        Shutdown(()),
+        #[bilrost(9)]
+        Error((bool, String)),
```

Comment:
I can bless this as a layout that should be on-the-wire identical to what i'm planning to do in the future to better support currently-unsupported `enum` types without requiring special conversion.